spec/support/contexts/odk_rendering.rb

```diff
@@ -0,0 +1,8 @@
+# odk rendering helper
+shared_context "odk rendering" do
+
+  # decorate objects
```

Comment:
Same here. `decorate(obj)` is already a good cue that this method decorates things.